app/src/main/java/net/aiscope/gdd_app/ui/main/MainActivity.kt

```diff
@@ -45,6 +43,14 @@ class MainActivity : AppCompatActivity(), SelectDiseaseView, LogoutFLow {
                 presenter.saveDisease(dropdown_select_disease.selectedItem.toString())
             }
         }
+
+        if(intent.hasExtra("SAMPLE_SAVED")) {
+            Snackbar.make(
```

Comment:
This is the reason that i says if it is better to have a custom component that wraps this logic... make sense?

Reply:
A new component has been created as mentioned above.